pkg/client/monitoring/v1alpha1/alertmanager.go

```diff
@@ -83,20 +82,20 @@ func (a *alertmanagers) Create(o *Alertmanager) (*Alertmanager, error) {
 func (a *alertmanagers) Get(name string) (*Alertmanager, error) {
 	obj, err := a.client.Get(name)
 	if err != nil {
-		return nil, fmt.Errorf("retrieving alertmanager failed: %s", err)
+		return nil, err
 	}
 	return AlertmanagerFromUnstructured(obj)
 }
 
 func (a *alertmanagers) Update(o *Alertmanager) (*Alertmanager, error) {
 	ua, err := UnstructuredFromAlertmanager(o)
 	if err != nil {
-		return nil, fmt.Errorf("unstructuring alertmanager failed: %s", err)
+		return nil, err
 	}
 
 	ua, err = a.client.Update(ua)
 	if err != nil {
-		return nil, fmt.Errorf("updating alertmanager failed: %s", err)
+		return nil, err
```

Comment:
I didn't feel it would add a lot more information to it here, and if we did wrap the error we would be requiring the user of this client to use `errors.Cause`, and therefore forcing a dependency on a user. I'd like to avoid that.